ctapipe/image/waveform_cleaning.py

```diff
@@ -67,6 +67,24 @@ def apply(self, waveforms):
         return waveforms
 
 
+class BaselineWaveformCleaner(WaveformCleaner):
+    """
+    Basic waveform cleaner that subtracts the waveform baseline
+    estimated as the mean waveform value in the interval [baseline_start,baseline_end]
+    """
+    baseline_start = Int(0, help='Start sample for baseline estimation').tag(config=True)
+
+    baseline_end = Int(10, help='End sample for baseline estimation').tag(config=True)
+
+    def apply(self, waveforms):
+        # Subtract baseline
+        baseline = waveforms - np.mean(
```

Comment:
Very minor note, but its not really "baseline" you obtain here, but rather "waveform_corrected" instead

Reply:
Hi Jason, actually it seems to me that the name an the description of this very simple and short function is enough clear.

I have now a question.  Who is supposed to accept PRs in ctapipe? For such simple type of PRs (reviewed by at least 3 persons) it would be correct to avoid long delay times.

Reply:
Let me be clearer. The variable you obtain on line 81 is currently called baseline. But it is not baseline, it is the waveform subtracted by the baseline, and is misleading.

Indeed, this review should be accepted very quickly.

Reply:
Ah, you are right, I misunderstood :-) 
Yes, l change it rapidly